pom.xml

```diff
@@ -73,7 +73,7 @@
     <cs.jasypt.version>1.9.2</cs.jasypt.version>
     <cs.trilead.version>1.0.0-build217</cs.trilead.version>
     <cs.ehcache.version>2.6.9</cs.ehcache.version>
-    <cs.gson.version>1.7.2</cs.gson.version>
+    <cs.gson.version>[2.4,)</cs.gson.version>
```

Comment:
Our standard currently is to pin to a specific version, in this case you depend on any 2.4.X version. Although I understand your reasons, is this the PR to change that behavior?

It makes testing harder when deps change as long as we don't have proper testing in place.